metasynth/var.py

```diff
@@ -72,6 +72,9 @@ def __init__(self,  # pylint: disable=too-many-arguments
         if self.prop_missing is None:
             raise ValueError(f"Error while initializing variable {self.name}."
                              " prop_missing is None.")
+        if self.prop_missing < -1e-8 or self.prop_missing > 1+1e-8:
```

Comment:
That looks like a very weird check. Why not:
```suggestion
        if self.prop_missing < -1.0 or self.prop_missing > 1.0:
```

is this to do with how the misprop is computed internally?

Reply:
I wanted to give it a little bit more leeway, since users might have their own computation of the prop_missing, so I don't want to put it at exactly 0.0 and 1.0. It's just to ensure that small rounding errors don't give errors.